ivy/functional/frontends/paddle/nn/functional/common.py

```diff
@@ -109,3 +109,8 @@ def interpolate(
 def linear(x, weight, bias=None, name=None):
     weight = ivy.swapaxes(weight, -1, -2)
     return ivy.linear(x, weight, bias=bias)
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+def alpha_dropout( x, p=0.5, training=True, name=None ):
+    return ivy.alpha_dropout(x, p=p, training=training)
```

Comment:
There is no `alpha_dropout` in Ivy API you will need a compositional implementation here 🙂